### DISCUSSION TITLE
streamlink: error: unrecognized arguments: --pluto-tv-skip-ads --pluto-tv-no-ads

### DISCUSSION THREAD

**Discussion by merrychrishna** [streamlink/streamlink#5163] [Q&A] [answered]
### Description

streamlink: error: unrecognized arguments: --pluto-tv-skip-ads --pluto-tv-no-ads

are these parameters supposed to work with the newest version of streamlink by default? because they are not working for me.

**Comment by gravyboat** [comment, marked as answer]
No. Pluto tv has no CLI options as per the docs: https://streamlink.github.io/plugins.html#pluto, https://streamlink.github.io/cli.html.
